### ISSUE TITLE
Website tooltip issues

### ISSUE THREAD

**Issue by natalia-muzyka** [ChildMindInstitute/mindlogger-web#133]
**1) There is no tooltip icon near the response option with tooltip.** 
Desktop: Tooltip is shown only after hovering the response option. 
Mobile devices: Tooltip is shown only after tapping on the response option image. 
Video: https://www.screencast.com/t/HbTTg2kfd3
Win 10 // Chrome 93.0.4577.63 
iPhone SE // iOS 14.2
Samsung S7 // Android 8.0

**2) [Mobile device][Portrait] The tooltip is trimmed on the small screen** 
Screenshot: https://www.screencast.com/t/EF9mcVln
iPhone SE // iOS 14.2
Samsung S7 // Android 8.0

**Environment:** 
https://web-staging.mindlogger.org/
maddison_ml@protonmail.com	// qwertyuio
applet: webapp test (1)
activity: radio, checkbox
Applet password: Qwe123!!!

**Comment by jj105** [comment]
I guess we were having deployment issue on web-staging when you were testing
It works fine on current web-staging

![2.png](https://images.zenhubusercontent.com/5e565f9bd9593e11ebab005a/96e516da-d069-4512-97b9-fd9cf1db2f91)
![1.png](https://images.zenhubusercontent.com/5e565f9bd9593e11ebab005a/45a767a8-6983-4e29-be87-b52740ab4f2b)

**Comment by natalia-muzyka** [comment]
Verified as fixed.

#1 video: https://www.screencast.com/t/v9gAvBGIW
#2 screenshot: https://www.screencast.com/t/RMNpWprAX

@jj105 
new issue: 
#3 On the Win 10 // Chrome 93.0.4577.63 the tooltip icon circle isn't rounded. On mobile device it is displayed with correct ration (in the circle)
https://www.screencast.com/t/XEEYsd30l

**Comment by jj105** [comment]
Hmm, it shows me as correct ratio on win 10 // Chrome 93.0.4577.63

![image.png](https://images.zenhubusercontent.com/5e565f9bd9593e11ebab005a/568857cb-fc17-4f19-9717-91791ec59142)

**Comment by natalia-muzyka** [comment]
@jj105 yes, looks as expected now for me as well:
https://www.screencast.com/t/b9wdV9QfM

**Comment by natalia-muzyka** [comment]
@jj105 I found when the tooltip is displayed on the desktop incorrectly: 
it happens only if the response option has more than one row:
![image.png](https://images.zenhubusercontent.com/60ab57c8916945b978a3da92/d5759687-fb4b-467b-b0b0-8c57e2eeaac2)

could you check, please, on this account and activity?:
maddison_ml@protonmail.com // qwertyuio
applet: webapp test (1)
activity: multiple

**Comment by natalia-muzyka** [comment]
Verified as fixed
![image.png](https://images.zenhubusercontent.com/60ab57c8916945b978a3da92/4c010199-7ac6-41ab-a53f-05a1e5840b06)

Environment:
https://web-staging.mindlogger.org/
Win 10 // Chrome 93.0.4577.82 
geria2.test@gmail.com	McgzF127
applet: webapp test
activity: multiple
